app/templates/main/validation-errors.html

```diff
@@ -0,0 +1,40 @@
+{% extends "base.html" %}
+
+{%- from 'govuk_frontend_jinja/components/file-upload/macro.html' import govukFileUpload -%}
+{%- from 'govuk_frontend_jinja/components/button/macro.html' import govukButton -%}
+{%- from "govuk_frontend_jinja/components/accordion/macro.html" import govukAccordion -%}
+{%- from "govuk_frontend_jinja/components/accordion/macro.html" import govukNotificationBanner -%}
+{%- from "govuk_frontend_jinja/components/table/macro.html" import govukTable -%}
+{%- from "errorTable.html" import errorTable -%}
+{%- from "errorSummary.html" import errorSummary -%}
+{%- from "help-links-dropdown.html" import helpLinksDropdown -%}
+
+{% block beforeContent %}
+{{ super() }}
+{% endblock beforeContent %}
+
+{% block content %}
+    <div class="upload-data-container govuk-!-margin-top-5">
+        <form method="post" action="{{ url_for('main.upload') }}" enctype="multipart/form-data">
+            <h2 class="govuk-heading-m">There are errors in your return</h2>
+            <p class="govuk-body">Fix these errors and re-upload your return.</p>
```

Comment:
This line is "The following errors have been found in your return." on main atm. The content here is what design are asking us to change too. I think the idea for this PR is to maintain all functionality from the current test env